File: library-beta/src/main/java/com/useek/library_beta/USeekPlayerCloseListener.java
package com.useek.library_beta;

/**
 * Created by Chris Lin on 11/4/2017.
 *
 * Interface to retrieve close fragment or activity
 *
 */

public interface USeekPlayerCloseListener extends USeekPlayerListener {

    /**
     * Called when user clicked close button to dismiss the Activity or Fragment.
     *
     * @param useekPlayerView   USeekPlayerView object of closed
     */
    void useekPlayerDidClosed(USeekPlayerView useekPlayerView);

}

File: library-kt-beta/src/main/java/com/useek/library_kt_beta/USeekPlayerCloseListener.kt
package com.useek.library_kt_beta

interface USeekPlayerCloseListener: USeekPlayerListener {
    /**
     * Called when user clicked close button to dismiss the Activity or Fragment.
     *
     * @param useekPlayerView   USeekPlayerView object of closed
     */
    abstract fun useekPlayerDidClosed(useekPlayerView: USeekPlayerView)

}